components/Navigation.js

```diff
@@ -1,18 +1,31 @@
 import Link from "next/link";
+import Image from "next/image";
+import { slide as Menu } from "react-burger-menu";
+
 export default function Navigation() {
   return (
-    <nav>
-      <ul>
-        <li>
-          <Link href="/">Home</Link>
-        </li>
-        <li>
-          <Link href="/create">New Project</Link>
-        </li>
-        <li>
-          <Link href="/favorites">Favorite Projects</Link>
-        </li>
-      </ul>
-    </nav>
+    <Menu
```

Comment:
Something seems to be not right with the burger menu. When I click on one of the Links, the Menu does not close automatically, which is something I would expect. This only works on the Favorite page. 
Make sure to check the library to see if there is something to force this behavior. Also the page stays grey and I have to click on the page outside of the menu to focus the page again, also not expected behaviour. Have a look to see how you can fix that before reposting this PR :)